.devcontainer/chefs_local/local.json.sample

```diff
@@ -55,13 +55,25 @@
         "public": {
           "limitApiKey": "120",
           "limitFrontend": "500",
-          "windowMs": "60000",
+          "windowMs": "60000"
         }
       },
       "encryption": {
         "proxy": "352f7c24819086bf3df5a38c1a40586045f73e0007440c9d27d59ee8560e3fe7"
       }
     },
+    "features": {
+      "eventStreamService": true
+    },
```

Comment:
If the feature flags are always going to be enabled so we can go straight to production with this, is it too much effort to remove the feature flag code now? Or would it be better to do it after this PR?

Reply:
it's a good question. proceeding directly to production negates the  (current) need for this. I'll take a look and see if it's simple enough to rollback. ideally we could remove the code as it is unused.

Reply:
removed feature flags.